snuba/cli/optimize.py

```diff
@@ -24,11 +24,15 @@
     required=True,
 )
 @click.option("--log-level", help="Logging level to use.")
+@click.option(
```

Comment:
are you sure we're going to cap out at 2? Why not just make the amount of parallelism a number?

Reply:
You can even validate the number not going higher than 2, but from an interface perspective I think this makes more sense